cli/run_util.go

```diff
@@ -552,12 +552,13 @@ func activateAndRunSteps(
 		}
 
 		stepResults := models.StepRunResultsModel{
-			StepInfo: stepInfoCopy,
-			Status:   resultCode,
-			Idx:      buildRunResults.ResultsCount(),
-			RunTime:  time.Now().Sub(stepStartTime),
-			ErrorStr: errStr,
-			ExitCode: exitCode,
+			StepInfo:  stepInfoCopy,
+			Status:    resultCode,
+			Idx:       buildRunResults.ResultsCount(),
+			RunTime:   time.Now().Sub(stepStartTime),
+			ErrorStr:  errStr,
+			ExitCode:  exitCode,
+			StartTime: startTime,
```

Comment:
why do not you use `stepStartTime` or the copy of `stepStartTime` if it would point to the last startTime?